libraries/preferences/api/src/main/kotlin/io/element/android/features/preferences/api/store/SessionPreferencesStore.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.preferences.api.store
+
+import kotlinx.coroutines.flow.Flow
+
+interface SessionPreferencesStore {
+    suspend fun setSendPublicReadReceipts(enabled: Boolean)
+    fun isSendPublicReadReceiptsEnabled(): Flow<Boolean>
+
+    suspend fun clear()
```

Comment:
Note: this is never called. Not a big deal but should be fixed later.

Reply:
Yeah, I need to figure out the right way to clean up this data when the user is logged out. I'll do it in another PR.